pkg/backend/openshiftcluster/delete.go

```diff
@@ -88,7 +88,7 @@ func (m *Manager) Delete(ctx context.Context) error {
 	m.log.Printf("deleting resource group %s", resourceGroup)
 	err = m.groups.DeleteAndWait(ctx, resourceGroup)
 	if detailedErr, ok := err.(autorest.DetailedError); ok &&
-		detailedErr.StatusCode == http.StatusForbidden {
+		detailedErr.StatusCode == http.StatusForbidden || detailedErr.StatusCode == http.StatusNotFound {
```

Comment:
```suggestion
		(detailedErr.StatusCode == http.StatusForbidden || detailedErr.StatusCode == http.StatusNotFound) {
```

^ isn't the above necessary due to operator precedence?

Reply:
Good catch! I didn't notice &&!